### ISSUE TITLE
Registering task while disabling

### ISSUE THREAD

**Issue by i0xHeX** [VolmitSoftware/React#459]
On disable:
```
[00:23:53] [React Monitor/WARN]: Exception in thread "React Monitor" org.bukkit.plugin.IllegalPluginAccessException: Plugin attempted to register task while disabled
[00:23:53] [Server thread/INFO]: Saving players
[00:23:53] [React Monitor/WARN]: 	at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftScheduler.validate(CraftScheduler.java:488)
[00:23:53] [React Monitor/WARN]: 	at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftScheduler.runTaskTimerAsynchronously(CraftScheduler.java:168)
[00:23:53] [React Monitor/WARN]: 	at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftScheduler.scheduleAsyncRepeatingTask(CraftScheduler.java:164)
[00:23:53] [React Monitor/WARN]: 	at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftScheduler.scheduleAsyncDelayedTask(CraftScheduler.java:138)
[00:23:53] [React Monitor/WARN]: 	at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftScheduler.scheduleAsyncDelayedTask(CraftScheduler.java:121)
[00:23:53] [React Monitor/WARN]: 	at com.volmit.react.util.RQ.run(RQ.java:55)
[00:23:53] [React Monitor/WARN]: 	at com.volmit.react.util.A.<init>(A.java:7)
[00:23:53] [React Monitor/WARN]: 	at com.volmit.react.util.TPSMonitor$3.<init>(TPSMonitor.java:171)
[00:23:53] [React Monitor/WARN]: 	at com.volmit.react.util.TPSMonitor.run(TPSMonitor.java:171)
```

React 6.577
Version 1.13.1 / Spigot / Paper

**Comment by i0xHeX** [comment]
* bump *
React 6.578
Version 1.13.2 / Spigot / Paper

**Comment by LogGits** [comment]
I am also getting this on paperspigot 1.12.2 build 1587.

**Comment by cyberpwnn** [comment]
This doesnt cause any issues, its just sampling.

**Comment by LogGits** [comment]
Can we get this error removed. It's the only error i see in 1.13 now

**Comment by cyberpwnn** [comment]
Sure, i'll have time to resolve it probably on sunday or monday. 

Also, i still need to run over R6 with 1.13 since some features do not work with it yet. 
You should know though, that this error literally has no impact on the server. 

React sampling runs async from the server, because of this, the second the plugin is told to disable, it doesnt get the hint until 50 (or more) milliseconds later. So, when it tries to return sampled data to RAI, and RAI is in the process of shutting down, bukkit throws that exception since you cannot spawn tasks while shutting down.

The "fix" you are asking for is simply to catch the exception, not print it, and move on. The plugin is disabling.

**Comment by cyberpwnn** [comment]
"Fixed" in 6.580

```java
try
{
	Bukkit.getScheduler().scheduleAsyncDelayedTask(Bukkit.getPluginManager().getPlugin("React"), a);
}

catch(Throwable e)
{
	// Hush
}
```
